src/api/moviesApi.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:4000';
+
+export const getMovies_api = (params) => {
+  const config = {
```

Comment:
Configs for different api methods is mostly copy-pasted. Extract the common part into the base config and method-specific props will be added with object destructuring. Something like

    const config = {
      ...baseConfig,
      extraProp: 'whatever',
    }